lib/game_collection.rb

```diff
@@ -1,5 +1,6 @@
 require_relative "game"
 require "csv"
+require "pry"
```

Comment:
should we have the require "pry" in our master?

Reply:
No I took it out. My bad